src/main/kotlin/blackjack/domain/Player.kt

```diff
@@ -0,0 +1,40 @@
+package blackjack.domain
+
+class Player(
+    val name: String,
+    private val hand: Hand = Hand(),
+    var state: State = State.HIT,
+) {
+    init {
+        updateState()
+    }
+
```

Comment:
```suggestion
    private val hand: Hand = Hand(),
) {
    var state: State = State.HIT
    private set
           
    init {
        updateState()
    }

```

초기 값을 준다면, 내부에서 활용해도 괜찮아보이네요 :)